### ISSUE TITLE
Unable To Find Arbitrator When Force Closing Channels

### ISSUE THREAD

**Issue by funk256** [lightningnetwork/lnd#2805]
### Background

I've been following this guide:  https://github.com/rootzoll/raspiblitz  I'm trying to force close all my channels to get my BTC back into my main wallet and ideally back to a different wallet away from lnd entirely.  When doing lncli closeallchannels --force     I get the usual info but this error on all of the channels it shows:

"error": "unable to close channel: rpc error: code = Unknown desc = unable to find arbitrator"

### Your environment

Bit inexperienced here.  On the page it has this info:  Version 1.1 with lnd 0.5.2-beta and bitcoin 0.17.0.1 -- truthfully I have not gone to version 1.1, I'm still on 1.0 which was released about a month ago.  Outside of that unclear how to check exactly what it's running.

### Steps to reproduce

lncli closeallchannels --force

I did do this for one channel as well:  lncli closechannel --force [info]     and as I recall it gave the same error.

### Expected behaviour

I was hoping each channel would be closed and I could transfer my funds elsewhere once they were verified on chain and back in my wallet.

### Actual behaviour

I get an error that I cannot find any documentation for.

Bonus question...I do have some funds sitting simply on my wallet and not locked up in a channel.  What is the command to send those funds out to another wallet completely unrelated to the lightning network?  Thanks for your help.

**Comment by wpaulino** [comment]
What do the logs show for the channels you're attempting to close?

> What is the command to send those funds out to another wallet completely unrelated to the lightning network?

See `lncli sendcoins -h`.

**Comment by funk256** [comment]
Great, was unaware of the -h command so that helps a lot.  Thank you for that.

Unfortunately I'm not entirely sure how to produce the logs you are referring to there.  I took a look at the help commands and did a quick Google without finding anything I thought might give you what you're looking for.  For full disclosure, I did get some solid help on the raspiblitz GitHub and the info from that is in this thread:  https://github.com/rootzoll/raspiblitz/issues/450  Maybe some of what you are referring to is in there or otherwise would be helpful.

Outside of that, if you have any commands you want me to run I will do my best to come back with anything that can help.

**Comment by wpaulino** [comment]
From looking at the thread you linked, it seems like you've run into disk corruption?

I've seen others run into this issue though, so there might be something here. You can most likely find your logs at `~/.lnd/logs/bitcoin/mainnet/`. You can then search these for the `channel_point` record and paste the results here.

**Comment by funk256** [comment]
Ah, darn.  I'm getting this when I paste that command into the console:  -bash: /home/admin/.lnd/logs/bitcoin/mainnet/: No such file or directory

I'm definitely going to try flashing the SD card and see where that takes me.  I'll report back if that solves the issue or with whatever happens.  Thanks.

**Comment by fluidvoice** [comment]
> Ah, darn. I'm getting this when I paste that command into the console: -bash: /home/admin/.lnd/logs/bitcoin/mainnet/: No such file or directory
> 
> I'm definitely going to try flashing the SD card and see where that takes me. I'll report back if that solves the issue or with whatever happens. Thanks.

That directory is wrong. Logs are located here: **/mnt/hdd/lnd/logs/bitcoin/mainnet/**

**Comment by halseth** [comment]
@funk256 What you can do (when you are able to locate the logs) is to restart `lnd`, attempt to force close one of the channels in question, and then post the logs from startup.

Even better if you can set `debuglevel=debug` in your config before starting lnd :)

**Comment by funk256** [comment]
Hey, just wanted to hop in and give an update.  Sorry for the slow reply.  I flashed a new version of the raspiblitz image to my SD card and that pretty much fixed everything.  Just so I'm not leaving anyone hanging...@fluidvoice I tried that directory as well and for whatever reason it tells me:  permission denied.

For @halseth, honestly not too sure how to do what you're describing there.  I certainly would like to learn more.  On the other hand, I'm happy it seems to be working at the moment so I'm a bit timid to tinker with it too much, lol.

Thanks so much for all the help, everyone.
